crates/evm/src/evm/error/rpc.rs

```diff
@@ -173,6 +173,14 @@ pub enum SignError {
     NoChainId,
 }
 
+// TODO: Fix https://github.com/chainwayxyz/citrea/issues/782
```

Comment:
Please remove this commented code and mark this PR as `Fixes #782` if this unconditional_recursion no longer presents in the code.